docs/content/drafts/UnderlineNav2.mdx

```diff
@@ -111,6 +111,18 @@ If there is still overflow, the component will behave depending on the pointer.
 </UnderlineNav>
 ```
 
+### Loading State for Counters
```

Comment:
In general, we use title case for headings.
```suggestion
### Loading state for counters
```

Reply:
I do appreciate these feedback! It adds up and bring great consistently 💖